### ISSUE TITLE
IDE0060 false positive (unused parameter)

### ISSUE THREAD

**Issue by mavasani** [dotnet/roslyn#32973]
**Version Used**:  `16.0.0 Preview 3.0 [28529.153.d16.0]`

**Steps to Reproduce**:

```csharp
class C
{
    public static bool M(out int x)
    {
        return LocalFunction(out x);

        bool LocalFunction(out int y)
        {
            y = 0;
            return true;
        }
    }
}
```

**Expected Behavior**:
No IDE0060

**Actual Behavior**:
```
Severity	Code	Description	Project	File	Line	Suppression State
Message	IDE0060	Remove unused parameter 'y'	ConsoleApp6	C:\Users\mavasani\source\repos\ConsoleApp6\ConsoleApp6\Program.cs	7	Active
```
